plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -482,57 +507,118 @@ protected String errorMessageForInsertNegativeDate(String date)
     }
 
     @Override
+    @Test(timeOut = 60_000, invocationCount = 4)
     public void testInsertRowConcurrently()
+            throws Exception
     {
-        // TODO Support these test once kudu connector can create tables with default partitions
-        throw new SkipException("TODO");
```

Comment:
Could you separate into two commits? 
1. Remove redundant overridden tests in Kudu
2. Implement concurrent tests in Kudu

Reply:
@ebyhr Thanks! Yeah that's much clearer. Just pushed an update :)